internal/applier/helm.go

```diff
@@ -63,7 +68,11 @@ func (h *Helm) Apply(ctx context.Context, contentFS fs.FS, ext *ocv1alpha1.Clust
 		return nil, "", err
 	}
 
-	rel, desiredRel, state, err := h.getReleaseState(ac, ext, chrt, values, labels)
+	post := &postrenderer{
+		labels: labels,
```

Comment:
Do we need all the objects to have all the labels? What's the minimum of what we need each object to have?

Seems like we should have explicit answers for "what labels do release secrets need?" and "what labels to managed objects need?"

Reply:
Need to figure out how to handle release secret, applyset, etc. labels vs object labels

Reply:
This should be addressed now